### ISSUE TITLE
Example websites to look at. 

### ISSUE THREAD

**Issue by TrishZwei** [TrishZwei/Trish-Chris-Collaboration#2]
Links of example websites we like and may be inspired to emulate. When Chris adds himself - this should get assigned to him.

**Comment by chris9840** [comment]
https://www.mahina.com/cruise.html?fbclid=IwAR1CkQYB_2mBsVBErY4d8ecWbyVI865c2xPqYGj7m8MXBQSE1pMWmi3hgMg
https://svdelos.com/?fbclid=IwAR3CXfQlymaoRTUqwBLsYRNjvSejoxb7FGNff71_lJ9TrfcN0eaJdcPFsOw

These are two nice layouts

**Comment by TrishZwei** [comment]
Cool. Ours will probably not be so complex. 

Tell me why you like these particular sites? Is it the colors? What is it that draws you to these sites?

I like SVDelos because it is a responsive design and a bit more modern. Mahina probably was developed in the early 2000's and it is not responsive.  If you don not know what I mean, open the site in a browser on your desktop/laptop and shrink the window. In Mahina you will see that the elements do not adjust and get cut off when the browser width shrinks while the  SVDelos site adjusts, nothing gets cut off. I noticed that SVDelos happens to be a WordPress website with WooCommerce. But we're going to use it for inspiration.

To focus on one thing at a time. what kind of content do you want to include?   Videos? Blogs? image galleries? To start we probably want to focus on one type of content and the homepage. Then we can expand from there.

### Styling:
I see bright banners across the top and a casual font.  The SVDelos site uses red-orange as its attention/call to action color. This is probably to contrast against the black/white/blue of the rest of the pages. 

I imagine blue/black/white are a good place to start as an overall color scheme. I think I'd like to experiment with a vibrant blue or green as a attention/cta color. What are your thoughts?

**Comment by chris9840** [comment]
Yes Mahina is quite old but I find it very easy to navigate that would be absolute number one advantage to it. Second though the colors being used are not very easy on my eyes and could be because my eyes aren't as good as they use to me. 

SVdelos I think what makes the website good is I feel it's more user friendly for all age groups and for people maybe that are new to to the Internet. I don't like website that are so advance that its tuff to navigate around them and a lot of the important stuff is lost because the user doesn't know how to find it.
I do kinda feel as I frequently visit there page it almost looks like it's to much stuff that has been added to the first page but then again they are trying to make sure everything is covered with the youtube channel and all there social media outlets because this is how they make there money to fund their sailing adventures. I guess what I am asking how can you make it not feel so jumbled up? I'm getting ahead of myself here lol.

The picture at the top just under all the drop down tabs makes you feel warm and welcoming to the website as these people share their adventures over the last 10 years. The color blue/black/white and Green/blue I like that because I feel those are warm and welcoming colors. I know everyone is different but some very bright colors for me sometimes isn't easy on my eyes but I know everyone is different.
